hilo.rb

```diff
@@ -1,26 +1,28 @@
+# Devtrain: Ruby Hi-Lo
+
 # predefined number of guesses allowed
 guesses = 6
 ## computer secretly picks number between 1 and 100 inclusive
 number = rand(1...101)
```

Comment:
So weird rails syntax things. `1...101` is 1 - 100 but `1..100` also is. The `...` is exclusive and `..` is inclusive, and I need to google it everytime because I forget which is which.